### DISCUSSION TITLE
Issue with the automatic1111 webui colab

### DISCUSSION THREAD

**Discussion by MasterDenis** [TheLastBen/fast-stable-diffusion#584] [General]
Hi, So basically I am getting this error when trying to load the webui from the colab, I do not get the error on the dreambooth version of the webui though.

Essentially I cannot load some extensions on that version while it is possible to do so in other colab versions such as the dreambooth one.

I made sure to reinstall everything.

![image](https://user-images.githubusercontent.com/61897713/203194786-1f51b68a-8cdb-456b-a75e-4135a7b085ce.png)

Thanks.

**Comment by TheLastBen** [comment]
Use `!pip install -U diffusers`

**Reply by MasterDenis** [reply]
I add it on the last cell somewhere on the start, right?

**Reply by TheLastBen** [reply]
create a new cell and run it, I will add the updated diffusers later so you won't have to install it everytime

**Reply by MasterDenis** [reply]
Thanks, it worked, also That would greatly help, for now I get this:

```
Error loading script: api.py
Traceback (most recent call last):
  File "/content/gdrive/sd/stable-diffusion-webui/modules/scripts.py", line 184, in load_scripts
    module = script_loading.load_module(scriptfile.path)
  File "/content/gdrive/sd/stable-diffusion-webui/modules/script_loading.py", line 11, in load_module
    compiled = compile(text, path, 'exec')
  File "/content/gdrive/sd/stable-diffusion-webui/extensions/sd_dreambooth_extension/scripts/api.py", line 14
    db_half_model: bool = False,
                               ^
SyntaxError: invalid syntax
```

**Reply by TheLastBen** [reply]
there is a syntax error in api.py, make sure to update the A1111 repo to get the fix

**Reply by MasterDenis** [reply]
> there is a syntax error in api.py, make sure to update the A1111 repo to get the fix

yeah I did, I also had issues lately with the v2 SD model, here is a screenshot:
![image](https://user-images.githubusercontent.com/61897713/204020034-c2ed4520-cec8-4f04-aaa4-696d7434f67a.png)

**Reply by TheLastBen** [reply]
That's a RAM error, free colab doesn't offer enough

**Reply by joanorsky** [reply]
.... any way to lower that requirement?

**Reply by TheLastBen** [reply]
I'm actively searching for a solution

**Reply by MasterDenis** [reply]
anything came up so far?
also the xformers script wasnt included for the above issue from my last test yesterday.

**Reply by MasterDenis** [reply]
> I'm actively searching for a solution
New Issue about v2.0
It Kept Happening within 12 hours that I tried more than once.
![image](https://user-images.githubusercontent.com/61897713/204126653-19559f47-449f-4998-94ca-8269737e6df4.png)

**Reply by MasterDenis** [reply]
PS: after deleting the whole SD folder and running the colab again with v1.5, I seem to be getting this error:
![image](https://user-images.githubusercontent.com/61897713/204128463-c7d144ed-8a59-444e-8150-7c107e6ed99f.png)
Basically not even the normal v1.5 can be used at this moment, perhaps something is wrong with the installation script?
It worked just fine 2 days ago or so.

Then I get this error when pointing to a specific model and rerunning:
![image](https://user-images.githubusercontent.com/61897713/204128920-02b06cbb-811b-4afc-a3cd-f750b46e53f8.png)
@TheLastBen

**Reply by TheLastBen** [reply]
A1111 keep changing their code, so some times the colab breaks

**Reply by MasterDenis** [reply]
> A1111 keep changing their code, so some times the colab breaks

Alright, 
can we get on the main colab something that is stable for now until they have things stable on their side too?
For now the colab does not work at all on either configuration.

**Reply by TheLastBen** [reply]
it's fixed now, just use the colab from the link

**Reply by nawnie** [reply]
> > A1111 keep changing their code, so some times the colab breaks
> 
> Alright, can we get on the main colab something that is stable for now until they have things stable on their side too? For now the colab does not work at all on either configuration.

make a note of the current commit, then you can just change the line that says !gitclone url for webui to 

... and use git checkout <COMMIT> to change to the right commit:

!git checkout 91f25642453 number will be diffrent

copy pasta from:https://stackoverflow.com/questions/7832770/how-to-get-certain-commit-from-github-project

** i know this is resolved but if add when it breaks again there will be an easy fix posted here :D

**Reply by MasterDenis** [reply]
> it's fixed now, just use the colab from the link

True, Also I saw your new colab for the low/free colab vram, there is an issue where the colab gradio interface will not show the resulted images.
Also, it may be a bit dirtier in terms of design so I wonder if it is possible to have similar results in the main one for the free/low versions of the colab memberships?
Thanks.

**Reply by MasterDenis** [reply]
Another issue on the colab is the support for extensions, for some reason it says no such file or directory but the file is there:
![image](https://user-images.githubusercontent.com/61897713/204605325-5f1a68d5-6169-4fcc-b6fc-090cca1efdf4.png)
I tried using the new depth map for img2img, this was on the main version (so 1.5).
